simple/README.md

```diff
@@ -0,0 +1,21 @@
+# Simplified setup for WSO2 kubernetes Enterprise Integrator
+
+## Contents
+* Prerequisites
+
+* Quick Start Guide
+
+## Prerequisites
+* Install [Kubernetes  Client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the steps provided in the following quick start guide.
+* An already setup Kubernetes cluster. If you are unfamiliar with this context, you can use [this guide](https://kubernetes.io/docs/setup/pick-right-solution/) to set up the cluster.
+* WSO2 subscribed users can run **wso2ei.sh** with latest updates by providing their subscription username and password. If you do not possess an active WSO2 subscription already, run **wso2ei_open.sh** which does not require subscription credentials.
+
+## Quick Start Guide
+1. Download simplified kubernetes setup for WSO2 Enterprise Integrator (either **wso2ei.sh** or **wso2ei_open.sh**).From this point forward the steps are being described for wso2ei.sh. If you have downloaded wso2ei_open.sh please substitute wso2ei.sh with wso2ei_open.sh for every command.
+
+1. In the command line, move into the directory where you have downloaded the simplified kubernetes-apim setup. (Usually, the file would be in the “Downloads” directory unless you have changed the default directory to somewhere else.)
+1. Provide permissions for the setup file to execute by running **chmod +x wso2ei.sh**
```

Comment:
All points are numbered as 1. Shall we fix this?

Reply:
<img width="969" alt="Screen Shot 2019-04-29 at 2 17 09 PM" src="https://user-images.githubusercontent.com/47025945/56885249-a0a8ed80-6a89-11e9-87ff-156704855d32.png">
this is how it looks at the end. (a Numbered list)

Reply:
Please refer https://help.github.com/en/articles/basic-writing-and-formatting-syntax#lists for odered list syntax.

Reply:
will do